frontend/app/components/lesson-item.hbs

```diff
@@ -1,28 +1,46 @@
 {{#link-to "course.module.lesson" @lesson.slug }}
-<div class="module-item {{if @available "green" "red"}}">
+  <div class="module-item {{if @available "green" "red"}}">
 
 
-  <img class="module-image" src="http://placehold.jp/20/{{@module.color}}/ffffff/200x140.png?text={{@lesson.name}}"
-    alt="">
+    <img class="module-image" src="http://placehold.jp/20/{{@module.color}}/ffffff/200x140.png?text={{@lesson.name}}"
+      alt="">
 
-  <div class="module-description">
+    <div class="module-description">
 
-    <h2>{{@lesson.name}}</h2>
+      <h2>{{@lesson.name}}</h2>
 
-    <div>{{@lesson.description}}</div>
-    <br>
+      <div>{{@lesson.description}}</div>
+      <br>
+
+      <hr>
+    </div>
+
+    {{#if me.isAuthenticated}}
+
+      {{#if @lesson.percentage.percent}}
+
+
+        <BsProgress as |pg|>
+          <pg.bar @showLabel={{true}} @roundDigits={{0}} @type="success" @value={{@lesson.percentage.percent}} />
+        </BsProgress>
+      {{else}}
+        <BsProgress as |pg|>
+          <pg.bar @showLabel={{true}} @roundDigits={{0}} @type="success" @value={{0}} />
+        </BsProgress>
+      {{/if}}
+    {{else}}
+      Not authed
+    {{/if}}
 
     <hr>
   </div>
 
-  {{#if me.isAuthenticated}}
-
   <BsProgress as |pg|>
     <pg.bar @showLabel={{true}} @roundDigits={{0}} @type="success" @value={{@module.progress}} />
   </BsProgress>
```

Comment:
Here module progress is rendered but the component is meant for lessons. 
We do not need this code